src/FlipCounter.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+const FlipCounter = props => {
```

Comment:
Why did you change it from React.Component to a functional component?

We never use functional components, because they do not provide refs and are hard to refactor.

Please, change back to React.Component

Reply:
And, because you are using functional component, Typescript does not have type informtion about your props and can not help you to check if the required props are passed.

We are using Typescript to type check the props of the component, so it will tell you what you need to pass to the component and what you have to pass.